lib/teamsnap/api.rb

```diff
@@ -73,6 +73,8 @@ def self.via(klass, method)
 
     def self.parse_error(resp)
       return "Object Not Found (404)" if resp.status == 404
+      return "Forbidden (403)" if resp.status == 403 && resp.body == ""
```

Comment:
Is there a reason why we are not returning `Forbidden (403)` when `resp.status == 403` without an empty body?